packages/bits/src/lib/select-v2/constants.ts

```diff
@@ -4,3 +4,5 @@ import { IOptionedComponent } from "./types";
 
 // Will be renamed in scope of the NUI-5797
 export const NUI_SELECT_V2_OPTION_PARENT_COMPONENT = new InjectionToken<IOptionedComponent>("NUI_SELECT_V2_OPTION_PARENT_COMPONENT");
+export const ANNOUNCER_OPEN_MESSAGE = (optionsLength: number) => `${optionsLength} options available`;
+export const ANNOUNCER_CLOSE_MESSAGE = "Dropdown closed";
```

Comment:
@DmytroPutrin, can you check whether the LivaAnnouncer supports i18n? If so, it would be good to go ahead and implement that or create a ticket to do that. Even if LiveAnnouncer doesn't support it, I think it would be good to have a ticket for researching how to internationalize all of our a11y support.

Reply:
As I could see no - cdk insert element which screen reader reads inside dom without internalization. I created ticked (NUI-2139) for this.